runtime/bcutil/cfreader.c

```diff
@@ -2528,12 +2539,13 @@ j9bcutil_readClassFileBytes(J9PortLibrary *portLib,
 	}
 
 	/* Make sure that following verification uses the class file version number */
+	U_32 vmVersionShifted = flags & BCT_MajorClassFileVersionMask;;
```

Comment:
@DanHeidinga I figured out the version issue I was having was because the vm version is replaced by the class file version prior to checking the <clinit> structure. What do you think of this fix with passing in a new variable with the vm version?